tools/chafa/chicle-media-loader.c

```diff
@@ -224,6 +226,20 @@ loader_vtable [LOADER_TYPE_LAST] =
         (gint (*) (gpointer)) chicle_heif_loader_get_frame_delay
     },
 #endif
+#ifdef HAVE_COREGRAPHICS
+    [LOADER_TYPE_COREGRAPHICS] =
+    {
+        "CoreGraphics",
+        (void (*)(void)) chicle_coregraphics_loader_new_from_mapping,
+        (gpointer (*)(gconstpointer)) NULL,
+        (void (*)(gpointer)) chicle_coregraphics_loader_destroy,
+        (gboolean (*)(gpointer)) chicle_coregraphics_loader_get_is_animation,
+        (void (*)(gpointer)) chicle_coregraphics_loader_goto_first_frame,
+        (gboolean (*)(gpointer)) chicle_coregraphics_loader_goto_next_frame,
+        (gconstpointer (*) (gpointer, gpointer, gpointer, gpointer, gpointer)) chicle_coregraphics_loader_get_frame_data,
+        (gint (*) (gpointer)) chicle_coregraphics_loader_get_frame_delay
+    },
+#endif
```

Comment:
Should be moved up before libheif like you suggested in the comments.

Reply:
moved ([7ef12bf](https://github.com/hpjansson/chafa/pull/307/commits/7ef12bfa99d861840596927d02cf3adee57b5e2a))